### DISCUSSION TITLE
How to control the plot window size in code

### DISCUSSION THREAD

**Discussion by diverger** [alandefreitas/matplotplusplus#270] [Q&A]
I can't find way to control the window size in code, are there methods to implement this with matplot++?

Thanks.

**Comment by alandefreitas** [comment]
These member functions

https://github.com/alandefreitas/matplotplusplus/blob/8dbea7d359f7b4f456bca7a6015c32b61ad728f4/source/matplot/core/figure_type.h#L222-L235
